components/app.js

```diff
@@ -45,8 +48,38 @@ class App {
         );
     }
 
-    start(){
+    createGrade(name, course, grade) {
+        console.log(name, course, grade);
```

Comment:
Remember to cleanup any `console.log`s before creating a pull request, especially to `master`. Production code shouldn't have any `console.log`s in it, the only exception being `console.error` for errors. `console.log`s are a great tool and it's really good you are using them to verify your data, just make sure to remove them once you are done with them. 😄